### ISSUE TITLE
Create .exe version of open_with_windows.py

### ISSUE THREAD

**Issue by IsaacSchemm** [darktrojan/openwith#137]
open_with_windows.py could probably be rewritten as a .NET executable (C# or VB), which would remove the need to install Python or use the command prompt. It would require the .NET runtime, but most Windows users already have it installed (via Windows Update.)

A potential downside would be the complexity of maintaining different scripts/apps in different programming languages.

If you're interested, maybe I could find time to give it a try.

**Comment by darktrojan** [comment]
It could, but I chose not to do so, because with Python the user can read the code and know exactly what is (and isn't) being run. Not that most of them will, but I'm not comfortable shipping binary components with my extensions.

**Comment by soul4soul** [comment]
It's my first time using openwith in general. The install process was not seemless for me. 

- I didn't have python installed
- I tried python3.x but I couldn't get it to work
- With python2.x I was able to get the python script to install with "python open_with_windows.py install"

At least running admin command prompt I was able to get this to install to "C:\Program Files (x86)\openwith".

I am in favor of a standard windows install process - an installer msi or exe which would deliever an exe. You made it clear this is not something you will do. How about a powershell script? It is native to windows and is plain text. You could make it act as an installer, have it run as an admin from anywhere and it will install the appropriate files to "C:\Program Files".

**Comment by darktrojan** [comment]
I have been looking at using PowerShell. However, the need to have *something* ready now, given that Firefox 57 is now in beta, means Python is the only option until I can get PowerShell working and tested.

**Comment by mbooga** [comment]
Currently working on a C port of open_with_windows.py
Should be really lightweight.
I'll let you know once I get something ready.

**Comment by darktrojan** [comment]
The latest beta contains a PowerShell script that works in the same way as the Python one. I've decided not to use it, but now that it exists (and having poured hours into it) I might as well ship it. Note that you can run whatever you like on your machine as long as it is wired up the same way as the one I ship. (Also if I change the way that happens it's not my problem.)

**Comment by soul4soul** [comment]
I got the powershell script working. By default when it found IE it didn't include quotes around the path. Luckily you have an option to manually edit the command. Adding quotes made everything work again. It's a shame this wont be supported. I have turned off updates on this extensions since I do not want the PS script to break. At least using this extension the way it is right now still makes my life easier!

I look forward to issue #50 being addressed or PS to be supported again in the future.
